Faking serial console in order to get rid of booting timeout.

polac opened the pull request:
This seems to be fixing the problem I have described in issu #1 

More discussion about the issue can be found here: https://bugs.launchpad.net/cloud-images/+bug/1829625

Commit:
Faking serial console in order to get rid of booting timeout.

mgruhler reviewed (approved):
Not quite sure what your issue is, but that sounds reasonable and is working.

Thank you for your contribution!

(I actually never intended/thought that this repo is something that would be used by other people. It is/was just an easy way for me to quickly setup reproducable build environments. Anyways, thank you :-) )

polac commented:
Just to document this futher:

Following was my issue:
```
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'ubuntu/focal64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'ubuntu/focal64' version '20201007.0.0' is up to date...
==> default: Setting the name of the VM: 2004_focal_noetic_amd64_default_1602360087368_13331
==> default: Fixed port collision for 22 => 2222. Now on port 2201.
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 22 (guest) => 2201 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2201
    default: SSH username: vagrant
    default: SSH auth method: private key
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

If you look above, you should be able to see the error(s) that
Vagrant had when attempting to connect to the machine. These errors
are usually good hints as to what may be wrong.

If you're using a custom box, make sure that networking is properly
working and you're able to connect to the machine. It is a common
problem that networking isn't setup properly in these boxes.
Verify that authentication configurations are also setup properly,
as well.

If the box appears to be booting properly, you may want to increase
the timeout ("config.vm.boot_timeout") value.
```
I just couldn't never get past this  ```Timed out while waiting for the machine to boot.``` until I added that one line as suggested here:  https://bugs.launchpad.net/cloud-images/+bug/1829625

Thanks for sharing your personal setup. It gave me a quick head start to my own project. "Sharing is caring."
